decision-records/0001-schema-improvements.md

```diff
@@ -0,0 +1,110 @@
+# Proposal for Schema Improvements
+
+- Date: 2024-10-22
+
+## Introduction
+
+This RFC proposes improvements to the current data schema to improve usability. Since changing the data structure entails publishing a new major version, which is costly for downstream dependencies, it is relevant to consider the entire vocabulary and structure to avoid later changes in the near future.
+
+## Context and Problem Statement
+
+### Confusion around the `writer` field
+
+The `writer` field is filled in with `service provider` in 32 out of 41 existing terms types, which does not add much value. Other values, such as `transportation operator (airline, railway, bus…)` (once), `paid-for goods or service provider` (once), `online marketplace` (twice), `search engine or intermediation service provider` (twice), and `intermediation service provider` (three times), only provide information about the industry in which the service operates, which is not the original purpose of the `writer` field.
+
+### Ambiguity in the `audience` field
+
+There is ambiguity between the audience of the document and the audience of the service. Currently, the `audience` field could be interpreted in these two ways.
+
+## Proposed solutions
+
+### Solution A
+
+#### Schema
+
+```json
+"<terms type name>": {
+  "defines": "...",
+  "used by": "...", // Default value: "service provider"
+  "addressed to": "...", // Default value: "end user"
```

Comment:
```suggestion
  "addressed to": "...", // Default value: "end user"
  "applies to": "...", // Optional
```

Reply:
It seems a bit redundant to me.

Reply:
These two fields serve different purposes. However, if they still seem too similar, I would prefer using `applicability` (or `applies to`) as it feels less restrictive and allows for broader, more detailed explanations. 

For example:

```
"Imprint": {
    "topic": "identification of content author and hosting service for official inquiries",
    …,
    "applicability": "Required in some regions, such as EU countries, to ensure transparency and provide contact information"
}
```

Or:

```
"Developer Terms": {
    "topic": "APIs and programmatic access to content",
    …,
    "applicability": "Restricted to developers or partners authorized to use the service's APIs or development tools"
}
```

Reply:
Decided in sync call: `applicability` sounds like a mix of jurisdiction, industry and audience. We won't apply it as such.